bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java

```diff
@@ -2,6 +2,10 @@
 
 import java.util.Map;
 
+import javax.mail.Message;
```

Comment:
If those changes are part of a later pull request, they don't need to be submitted in this one. Please remove the from this pull request.